File: src/codes/Users_Manual/modulation_loop.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
let m = autd3::modulation::Sine::new(150. * Hz)
            .with_loop_behavior(LoopBehavior::infinite()); // infinite loop
let m = autd3::modulation::Sine::new(150. * Hz)
            .with_loop_behavior(LoopBehavior::finite(10).unwrap()); // 10 times
let m = autd3::modulation::Sine::new(150. * Hz)
            .with_loop_behavior(LoopBehavior::once()); // equivalent to finite(1).unwrap()
# } 

File: src/codes/Users_Manual/modulation_loop.cpp
//~#include<autd3.hpp>
//~int main() {
//~{
const auto m =
    autd3::modulation::Sine(150 * autd3::Hz)
        .with_loop_behavior(autd3::LoopBehavior::Infinite);  // infinite loop
//~}
//~{
const auto m =
    autd3::modulation::Sine(150 * autd3::Hz)
        .with_loop_behavior(autd3::LoopBehavior::Finite(10));  // 10 times loop
//~}
//~{
const auto m = autd3::modulation::Sine(150 * autd3::Hz)
                   .with_loop_behavior(
                       autd3::LoopBehavior::Once);  // equivalent to finite(1)
                                                    //~}
                                                    //~return 0; }